iceoryx_hoofs/include/iceoryx_hoofs/cxx/optional.hpp

```diff
@@ -185,6 +170,8 @@ class optional : public FunctionalInterface<optional<T>, T, void>
 
     /// @brief Will return true if the optional contains a value, otherwise false.
     /// @return true if optional contains a value, otherwise false
+    // AXIVION Next Construct AutosarC++19_03-A13.5.3: designed implementation shall be as close
+    // as std::optional and it requires operator bool
```

Comment:
I think the justification is still not precise enough. Can you please copy @elfenpiff's suggestion?

Reply:
@FerdinandSpitzschnueffler Have updated the justification